web-registry/src/components/organisms/EntityDisplayForm.tsx

```diff
@@ -46,6 +46,7 @@ export interface EntityDisplayValues {
   'http://regen.network/landSteward'?: DisplayValues;
   'http://regen.network/projectDeveloper'?: DisplayValues;
   'http://regen.network/projectOriginator'?: DisplayValues;
+  'regen:projectDeveloper'?: DisplayValues;
```

Comment:
@blushi I noticed the new standard uses this format `'regen:projectDeveloper'`, although we were using `'http://regen.network/projectDeveloper'` before. How do you think we should handle this right now?
 I was hoping to avoid increasing the scope and redoing all of the form input properties, but maybe it is unavoidable? I would also have to make sure all the existing project metadata was updated in PROD.

Reply:
these are the same things, the only difference is that `regen:projectDeveloper` corresponds to the compacted JSON-LD document form where we define "regen": "http://regen.network/" in the `@context` (see docs https://json-ld.org/spec/latest/json-ld/#compacted-document-form) 
I guess for now let's just use the expanded version in the metadata (eg `'http://regen.network/projectDeveloper'`) so that we don't have to touch this for now.
Later on, we might think of using https://github.com/digitalbazaar/jsonld.js/ so we can just work from same document form in the code without necessarily changing the underlying data (we do already have an issue to track that: https://github.com/regen-network/regen-registry/issues/724)

Reply:
Thanks. Using expanded format for now.